backup-manager/symfony/2.1/config/packages/backup_manager.yaml

```diff
@@ -0,0 +1,9 @@
+# See full config at https://github.com/backup-manager/symfony
+bm_backup_manager:
+    database:
+        development:
+            dsn: '%env(DATABASE_URL)%'
+    storage:
+        local:
+            type: Local
+            root: "%kernel.cache_dir%/db"
```

Comment:
Use single quotes here too?

Reply:
By the way, using the cache directory (which may be cleared frequently) is probably not a good idea, is it?

Reply:
You would probably configure a "off site location" for your backup. This is a configuration that will always work and that the user would probably use this config when testing the bundle's functinality.